src/EditorFeatures/CSharpTest2/Recommendations/DelegateKeywordRecommenderTests.cs

```diff
@@ -235,16 +236,22 @@ public async Task TestNotAfterSealed()
             => await VerifyAbsenceAsync(@"sealed $$");
 
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        public async Task TestNotAfterStatic()
-            => await VerifyAbsenceAsync(@"static $$");
+        public async Task TestAfterStatic()
+        {
+            await VerifyAbsenceAsync(SourceCodeKind.Regular, @"static $$");
+            await VerifyKeywordAsync(SourceCodeKind.Script, @"static $$");
```

Comment:
why absent in srtipt?

Reply:
They're allowed in script, not in regular. On the top level, `static` can't be a type.